tutorial/debezium-cassandra-init/cassandra.yaml

```diff
@@ -0,0 +1,1021 @@
+# Cassandra storage config YAML
```

Comment:
Instead of copying the entire Cassandra configuration file, could we also take the default one and just replace/amend the values we actually need to customize, e.g. using `sed`? I'm concerned about maintenance effort of this, e.g. when it comes to upgrading to a newer version. Right now it's hard to see what configuration changes are needed in this file.

Reply:
There were particularly 8-9 modifications that were needed to the original config file. This was @jpechane's idea of putting our customized configuration file. I don't think it will be trouble in upgrading to newer versions. @jpechane any points to add?

Reply:
@gunnarmorling Yes, that was my recommendation. As Cassandra is mostly developed in 4.x tree I don't suppose any important changes in 3.x default config file

Reply:
> As Cassandra is mostly developed in 4.x tree I don't suppose any important changes in 3.x default config file

That's my point though. When we update to 4.x at some point in the future, we'll likely bring in its default configuration file, just to make sure we don't miss any values they may have newly introduced. But then we need to re-apply our own changes, and it's hard to find out which those were. You'll have to go back to a default 3.x file and compare what we've changed.

If you don't warm up for the sed approach, then let's at least split up the commit into two:

* The first which adds the unmodified configuration file and everything else
* The second which does the required changes to the configuration file

That way we'll easily know what has to be adjusted whenever we need to update that file.

Reply:
@gunnarmorling Or even better consolidate all changes at the and of file and mark them as scuh

Reply:
@gunnarmorling @jpechane I think we need to use `sed`. Because we have to enable as well as disable properties from the configuration file. I just checked these are these modifications are to be done.
```
1. Remove enable_sasi_indexes
2. Remove replica_filtering_protection (cached_rows_warn_threshold: 2000 cached_rows_fail_threshold: 32000)
3. Add commitlog_directory
4. Add hints_directory
5. Add cdc_raw_directory
6. Add saved_caches_directory
7. Add data_file_directories 
8. Change enable_cdc to true
9. Change IPs from 172.18.0.4 to 172.0.0.1 in multiple places.
```

Reply:
@ani-sha, ok, but the "separate commit" strategy  should work, too, right? I don't mind too much which way tbh. Just assume you'd have to look into this in 6 months from now: how can you prepare future Anisha to know what needs to change at that point :)

Reply:
Point. :sweat_smile:  @gunnarmorling, so I am adding the unchanged config file in this commit. Then add the required changes in a separate commit.

Reply:
Ok, thanks, looking great now!